apps/yara/mapper.h

```diff
@@ -317,6 +324,9 @@ struct Mapper
 
     typename Traits::TCigar             cigars;
```

Comment:
Now we can rename suboptimalMatchesSet --> matchesSet, cigars --> cigarString, cigarSet --> cigars, suboptimalCigars --> cigarsSet.

You can copy these definitions for cigar-based variables:
```
    typename Traits::TCigarString       cigarString;
    typename Traits::TCigarsSet         cigarsSet;
    typename Traits::TCigars &          cigars;
```
Note that ```cigars``` is a reference now.